spinalcordtoolbox/download.py

```diff
@@ -159,7 +159,13 @@
             "https://github.com/ivadomed/multimodal-registration/releases/download/r20220512/models.zip"
         ],
         "default_location": os.path.join(__sct_dir__, "data", "deepreg_models"),
-    }
+    },
+    "data_normalize_metric_pam50": {
```

Comment:
> Question for the reviewer(s): is `data_normalize_metric_pam50` the name we want to use for this dataset?

Some nitpicks:

- Elsewhere in `sct_download_data`, we use `PAM50` (capitalized), so it might be good to unify that.
- Part of me wants to use a name that represents a noun, similar to the other dataset names? For example, rather than "data normalize metric pam50", we would say "PAM50-normalized metrics", which could then become the dataset name `PAM50_normalized_metrics` .

Reply:
I am fine with renaming the repo and folder!